Exercises/numbers/numbers.js

```diff
@@ -1,3 +1,30 @@
+//1. Assign by hand same value to 3 variables in binary, octal and hexadecimal format.
+const binaryNumber = 0b110111;
+const octalNumber = 0o67;
+const hexadecimalNumber = 0x37;
+
+console.log(binaryNumber, octalNumber, hexadecimalNumber);
+
+// 2. Implement function `extractNumbers` which will leave only numbers in passed string.
 function extractNumbers(text) {
-  return null;
+  const number = parseInt(text, 10);
+  if (isNaN(number) === true) {
+    return console.log('Text should starts with numbers');
+  }
+  return console.log(number);
```

Comment:
if you will do `return console.log()` it will return undefined, as `console.log` is function which returns `undefined` so just return number, or text